test/gurka/test_search_filter.cc

```diff
@@ -17,6 +17,7 @@ class SearchFilter : public ::testing::Test {
     |   2     |
     |         ↑
     |1    4  3|
+    |         |
```

Comment:
previously the 1 node was pretty much equidistant from either edge, so it was a toss up depending on precision. once precision was modified the toss up went the other way. this change makes it not a toss up